website/docs/d/account_groups.html.markdown

```diff
@@ -0,0 +1,48 @@
+---
```

Comment:
Add subcategory refers others docs and add an entry in allowd_categories.txt
For all docs

Reply:
Done